api/schemas/input/file.json

```diff
@@ -3,7 +3,7 @@
   "type": "object",
   "properties": {
     "name":           { "type": "string" },
-    "type":           { "type": "string" },
+    "type":           { "type": ["string", "null"] },
```

Comment:
Is this change supposed to be here?
A similar change was proposed in #288 and we eventually decided against it.